include/util/range_table.hpp

```diff
@@ -139,6 +139,18 @@ template <unsigned BLOCK_SIZE, bool USE_SHARED_MEMORY> class RangeTable
         sum_lengths = lengths_prefix_sum;
     }
 
+    void WriteARangeTable(osrm::storage::io::FileWriter &filewriter)
```

Comment:
Hm the naming here and below are a bit weird. The type is already `RangeTable`, so `Read` or `Write` is good enough.